faqs/galaxy/tools_help.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Where is the tool help?
+area: tools   
+description:  Finding tool support
+box_type: tip
+layout: faq    
+contributors: [jennaj, beachyesh]
+---
+There is documentation available on the tool form itself which mentions the following information:
+- Parameters
+- Expected format for input dataset(s)
+- Links to publications and ToolShed source repositories
+- Tool and wrapper version(s)
+- 3rd party author web sites and documentation
+
+Many tools include "most common" and "full parameter" run-time options. Scroll down on the tool form to locate:
```

Comment:
the first sentence is not clear to me... and the second one looks like the actual only step of the FAQ

Reply:
Yes, I basically took it verbatim from the old FAQ, and it doesn't fit well. I removed that part and made the single step in this FAQ bold.